packages/@ama-sdk/core/src/plugins/mgw-mdw-auth/mgw-mdw-auth.request.ts

```diff
@@ -27,15 +29,17 @@ export async function sha256(value: string) {
  * Generates hash-based message authentication code using cryptographic hash function HmacSHA256 and the provided
  * secret key
  * Should only be in a NodeJS MDW context
- *
  * @param value Value to hash
  * @param secretKey Secret cryptographic key
  */
-export function hmacSHA256(value: string, secretKey: string) {
+export async function hmacSHA256(value: string, secretKey: string) {
```

Comment:
I assume we can accept this breaking change because it was not exposed in the first place ?